### DISCUSSION TITLE
How can I open web links in the browser  but not inside the code-server windows?

### DISCUSSION THREAD

**Discussion by smbus2kw** [coder/code-server#4798] [Q&A]
Now I want to open a web link from the WebviewPanel  of code-server extension , but the web page was opened inside of code-server window (WebviewPanel ),  but not in the browser which code-server was displayed.  how can I open web links in the browser  but not inside the code-server windows?

**Comment by lg-nineoneninethreenine** [comment]
Maybe you right-click and check for Open in Browser? I don't know if this does exist, so I am unsure, try that though.

**Reply by jsjoeio** [reply]
Yeah this should work.

Otherwise, can you post a screenshot so we can see?
